.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+name: OctoDNS Documentation
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+  pull_request:
+    branches: [main, master]
```

all the octoDNS repos use `main`

```suggestion
    branches: [main]
```